### PR TITLE
[DOCS] Adds warning about exceptions requiring mappings

### PR DESCRIPTION
**Pull request by rylnd**
### [Preview of Exceptions page](https://security-docs_2110.docs-preview.app.elstc.co/guide/en/security/master/detections-ui-exceptions.html)

This PR is meant to address https://github.com/elastic/kibana/issues/133078 to a degree, by calling out that mappings are required for exceptions to function. While https://github.com/elastic/kibana/issues/133078 is not a bug, it is an unexpected behavior (for which we've received some SDHs), and this note is an attempt to make that ... less unexpected.

I believe that the majority of the necessary info is here, but I'd appreciate a review of copy/syntax etc. Not sure if this fits docs' guidelines and/or is legible.

### CODE REVIEW

**Comment by rylnd** [comment]
Note: since this has always been the behavior of exceptions, it'd be nice to backport this note to previous docs versions as well.

**Review by nastasha-solomon — COMMENTED** [review]
Thanks for filing this @rylnd ! Just left one comment for your consideration. My last two comments are minor editorial nits and I directed them towards Joe since he'll have the answer for them.

File: docs/detections/detections-ui-exceptions.asciidoc [hunk 1]
```diff
@@ -167,6 +159,16 @@ Like detection rule exceptions, you can add Endpoint agent exceptions either by
 
 You can also add Endpoint exceptions to rules that are associated with {elastic-endpoint} rule exceptions. To associate rules, when creating or editing a rule, select the <<rule-ui-advanced-params, *{elastic-endpoint} exceptions*>> option.
 
+[IMPORTANT]
+=====
+When you add an exception to the
+<<endpoint-rule-exceptions, Elastic Endpoint Security>> rule, you can select to
+add the exception to the Endpoint. When selected, the exception is added to
+both the detection rule *and* the Elastic Endpoint agent on your hosts.
```
[the remark this answers is not in the document]
Comment (jmikell821) [reply docs/detections/detections-ui-exceptions.asciidoc:167]: @nastasha-solomon  yes, should be `:elastic-endpoint:`.

[the remark this answers is not in the document]
Comment (nastasha-solomon) [reply docs/detections/detections-ui-exceptions.asciidoc:167]:
```suggestion
both the detection rule *and* the {elastic-endpoint} agent on your hosts.
```

**Review by joepeeples — COMMENTED** [review]
LGTM, with @nastasha-solomon's edits

**Review by yctercero — APPROVED** [review]

**Comment by rylnd** [comment]
@Mergifyio backport 7.9

**Comment by rylnd** [comment]
https://github.com/Mergifyio backport 7.10 7.11 7.12 7.13

**Pull request merged**